doc/README.md

```diff
@@ -0,0 +1,402 @@
+- Disable pincode on the game server. (Not supported yet)
+- Disable packet_obfuscation on the game server. (Not supported yet, causes invalid packets)
+
+## 3.5 Setup: RoBrowser
+
+First, you will need the project in your machine. You can fork your own version or just clone the repository by typing:
+
+```
+git clone https://github.com/MrAntares/roBrowserLegacy.git
+```
+
+Or you can just download our latest release [here.](ttps://github.com/MrAntares/roBrowserLegacy/archive/refs/heads/master.zip)
+
+Having the project extracted in your machine, you will need to install the node dependencies by running:
+
+```shell
+npm install
+```
+
+
+# 4. Serving and Running roBrowser
+
+Now we have everything to make our game run as expected. Before start, we have to start a web-server to host all our roBrowserLegacy. By default, our project already have one installed on the node dependencies, which is called `live-server`, configured under the project's [package.json](../package.json) pre-configured commands.
+
+Running the live server (browserless): 
+```bash
+npm run live
+
+# robrowser@1.0.0 live
+# > live-server . --no-browser
+# Serving "." at http://127.0.0.1:8000
+```
+
+Or if you're using PHP or Ruby you can type any of these commands:
+
+```shell
+php -S 0.0.0.0:8000
+
+# or
+
+ruby -run -ehttpd . -p8000
+```
+
+With any of them, you will be able to access `http://localhost:8000` with a proper web-server running locally. 
+
+## 4.1 Serving Game: Live Server/Production
+
+There's two builds available today: Development and Production. 
+
+This step/section is only recommended for a "Live" server. It will only pack all the resource files into one file to speed up loading. 
+
+First, we have to  access the **roBrowserLegacy** project root:
+
+```bash
+cd path/to/robrowserlegacy
+```
+
+Then you can simply run:
+```bash
+npm run build -- -O -T -H
+```
+
+It will generate 3 files inside `dist/Web`, which are **Online.js**, **ThreadEventHandler.js** and **index.html** that you have to move/copy into your **project root**.
+
+```
+cp dist/Web/* .
+```
+
+Now you're good to open your browser at `https://127.0.0.1:8000` and see your roBrowser base client running!
+
+> In case you want to minify your roBrowser, go on the instance config and turn the `development` flag into `false`.
+
+## 4.2 Serving Game: Development
+
+![RO Browser running in Chromium](img/ro-browser-dev.png)
+
+If you want to contribute directly on RoBrowserLegacy source, you should focus on `/src` path.
+
+
+First, go to your roBrowserLegacy Folder:
+
+```shell
+cd path/to/robrowserlegacy
+```
+
+You can start the development build by typing:
+
+```shell
+npm run serve
+# > robrowser@1.0.0 serve
+# > live-server ./dist/Web
+#
+# Serving "./dist/Web" at http://127.0.0.1:8080
+```
+
+Since we're under development purposes (modifying the source/testing) change in the roBrowser config inside the `dist/Web/index.html` with: `development: true`.
+
+
+## 4.3 Serving Game: using Browser
```

Comment:
i would move this section under "4.1 Serving Game: Live Server/Production" as it is another way to build `online.js` an `ThreadEventHandler.js`

Maybe section "4.1" need 2 sub section _"build using node"_ _"build using browser tool"_ or something like that

Reply:
I'm not sure if I understood what I'm suppose to do. 

Could you give an example on this 4.1.x?